ush/hpssarch_gen.sh

```diff
@@ -598,9 +599,9 @@ if [[ ${type} == "enkfgdas" || ${type} == "enkfgfs" ]]; then
           fi
         fi
         {
-          echo "${COM_ATMOS_ANALYSIS_MEM/${ROTDIR}\//}/${head}atmf00${FHR}.nc"
+          echo "${COM_ATMOS_HISTORY/${ROTDIR}\//}/${head}atmf00${FHR}.nc"
```

Comment:
```suggestion
          echo "${COM_ATMOS_HISTORY_MEM/${ROTDIR}\//}/${head}atmf00${FHR}.nc"
```

Reply:
Works in testing. Accepting commit suggestion.